kiwi/runtime_checker.py

```diff
@@ -587,3 +587,34 @@ def check_mediacheck_only_for_x86_arch(self):
                 raise KiwiRuntimeError(
                     message_tool_not_found.format(name=tool)
                 )
+
+    def check_preferences_data(self):
```

Comment:
Can we make the method name more explicit. Something like

```
check_minimal_required_preferences
```

or similar ?

Reply:
sure I am fine making this name more specific